https_client.cpp

```diff
@@ -19,7 +19,7 @@
 namespace net = boost::asio;
 
 #ifdef _WIN32
-namespace ssl = net::windows_sspi;
+namespace ssl = boost::windows_sspi;
 #else
 namespace ssl = net::ssl;
```

Comment:
There is no `net::ssl` in the Networking TS btw

Reply:
I guess I should use `namespace ssl = boost::asio::ssl` instead then?

Reply:
That would be ok